### DISCUSSION TITLE
website's code location

### DISCUSSION THREAD

**Discussion by goyalyashpal** [lavv17/lftp#696] [General]
Is the website https://lftp.yar.ru Open source? so as to ease the process of making suggestions/prs.

i also wanted to make a suggestion to home page, and this discussion does too: https://github.com/lavv17/lftp/discussions/684

To be clear: i am not suggesting to host the website here on github itself, i am fine with it being anywhere.

**Comment by nzec** [comment]
The website hasn't been updated for a while either.
